app/command.go

```diff
@@ -290,6 +384,16 @@ func (a *App) tryExecuteCustomCommand(args *model.CommandArgs, trigger string, m
 
 	p.Set("trigger_id", args.TriggerId)
 
+	userMentionMap := a.mentionsToTeamMembers(message, team.Id)
```

Comment:
I'm sure this is still in-progress, but let's be sure to extend the unit tests to cover this.

Reply:
On it right now! :)

Reply:
Done in https://github.com/mattermost/mattermost-server/pull/13762/commits/552c67e6d459e98b646fd149c792a2bb0eb7486d. I tested `mentionsToTeamMembers` and `mentionsToPublicChannels`, as well as the `ToURLValues` functions. While it's true that the loop adding the `url.Values` to `p` here is not explicitely tested, I though it was not worth to build an integration test for the function that created a slash command that reported the `url.Values` read back in its response, as the code inside that test would be as untested as the loop here. So I kept it simple and trusted that the thorough testing in the functions generating the values is enough.